packages/google_sign_in/google_sign_in/README.md

```diff
@@ -139,6 +147,78 @@ Future<void> _handleSignIn() async {
 }
 ```
 
+In the web, you should use the **Google Sign In button** (and not the `signIn` method)
+to guarantee that your user authentication contains a valid `idToken`.
+
+For more details, take a look at the
+[`google_sign_in_web` package](https://pub.dev/packages/google_sign_in_web).
+
+## Working with scopes, and incremental authorization.
+
+If your app supports both mobile and web, read this section!
+
+### Checking if scopes have been granted
+
+Users may (or may *not*) grant all the scopes that your application requests at
+Sign In. In fact, in the web, no scopes are granted by signIn or silentSignIn anymore.
+
+Your app must be able to:
+
+* Detect if the authenticated user has authorized the scopes your app needs.
+* Detect if the scopes that were granted a few minutes ago are still valid.
+
+There's a new method that allows your app to check this:
+
+```dart
+final bool isAuthorized = await _googleSignIn.canAccessScopes(scopes);
+```
+
+(Only implemented in the web from version 6.1.0)
+
+### Requesting more scopes when needed
+
+If your app determines that the user hasn't granted the scopes it requires, it
+should initiate an Authorization request **from an user interaction** (like a
```

Comment:
This user action restriction is web-specific, right?

Reply:
AFAIK yes. Noted in the docs.